lib/rollup/tasks/totals.ex

```diff
@@ -0,0 +1,61 @@
+defmodule Mix.Tasks.Castle.Rollup.Totals do
```

Comment:
Converted the old rollup task to the new cron-style.  Will eventually just nuke this, when the controllers start pulling from postgres instead.